assets/agenda/utils.ts

```diff
@@ -1015,42 +1023,54 @@ export const getCoverageTooltip = (coverage: any, beingUpdated?: any) => {
     const slugline = coverage.item_slugline || coverage.slugline;
     const coverageType = getCoverageDisplayName(coverage.coverage_type);
     const coverageScheduled = moment(coverage.scheduled);
+    const assignee = getCoverageAsigneeName(coverage) || '';
+    const desk = getCoverageDeskName(coverage) || '';
 
     if (coverage.workflow_status === WORKFLOW_STATUS.DRAFT) {
-        return gettext('{{ type }} coverage {{ slugline }} {{ status_text }}', {
+        return gettext('{{ type }} coverage {{ slugline }} {{ status_text }}, assignee: {{assignee}}, desk: {{desk}}', {
```

Comment:
How does this look when `assignee` or `desk` is not actually defined? Would this needlessly display
`assignee: , desk: `